hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/JettyMessageSender.java

```diff
@@ -22,13 +22,13 @@
     private final HttpClient client;
     private final ResolvableEndpointAddress endpoint;
     private final long timeout;
-    private final DefaultHttpTraceIdAppender traceIdAppender;
+    private final DefaultHttpMetadataAppender metadataAppender;
```

Comment:
I think it would be nice if we use an interface MetadataAppender<Request> here instead of concrete (and default) implementation